dev/conductor/bin/conductor

```diff
@@ -37,4 +37,9 @@ BIN_DIR="$(cd "${PROG_NAME%/*}" ; pwd -P)"
 REPO_DIR="$BIN_DIR/../../.."
 DART_BIN="$REPO_DIR/bin/dart"
 
+# Ensure pub get has been run in the repo before running the conductor
```

Comment:
I think you might as well always run it, since the package config could always be stale no?

Reply:
True, I considered that possibility. I was worried about latency, but this tool is mostly waiting for git commands to run, so SGTM